WalletWasabi/Legal/LegalDocuments.cs

```diff
@@ -10,6 +10,7 @@ namespace WalletWasabi.Legal;
 public class LegalDocuments
 {
 	public static readonly string EmbeddedFilePath = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), "Legal", "Assets", "LegalDocuments.txt");
+	public static readonly string EmbeddedFilePathForWW2 = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), "Legal", "Assets", "LegalDocuments2.txt");
```

Comment:
Since I don't know what the file name is going to be, I went with the simplest.

This going to change later on.